gnd/src/main/java/com/google/android/gnd/ui/home/HomeScreenFragment.java

```diff
@@ -120,6 +130,14 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
     viewModel.getShowAddFeatureDialogRequests().observe(this, this::onShowAddFeatureDialogRequest);
     viewModel.getFeatureSheetState().observe(this, this::onFeatureSheetStateChange);
     viewModel.getOpenDrawerRequests().observe(this, __ -> openDrawer());
+
+    showFeatureDialogRequests = PublishSubject.create();
+
+    Observable<Feature> featureStream =
+        showFeatureDialogRequests.flatMapMaybe(
+            fragmentManager -> addFeatureDialogFragment.show(fragmentManager));
+
+    featureStream.as(autoDisposable(this)).subscribe(viewModel::addFeature);
```

Comment:
Since `featureStream` is only used here, it might be slightly more fluent to just change this with its above declaration.

Reply:
Indeed! Done

Reply:
Done!